toml-to-ical/src/lib.rs

```diff
@@ -113,7 +113,7 @@ impl DateTimeExt for NaiveDate {
 pub trait External {
     type Error;
 
-    fn from_path(path: &Path) -> Result<Calendar, Self::Error>;
+    fn load_calendar(&self, path: &Path) -> Result<Calendar, Self::Error>;
```

Comment:
```suggestion
    fn load_without_includes(&self, path: &Path) -> Result<Calendar, Self::Error>;
```

Just to make it clearer that `Calendar::load` should be preferred

Reply:
OK.  Updated to this.